packages/shared/lib/core/app/interfaces/platform-event-map.interface.ts

```diff
@@ -0,0 +1,23 @@
+import { DashboardRoute } from '@core/router'
+
+import { IAppUpdateDownloadProgress } from './app-update-download-progress.interface'
+import { IAppVersionDetails } from './app-version-details.interface'
+
+export interface IPlatformEventMap {
+    'menu-logout': void
```

Comment:
just as a comment, these are very desktop events, we might want to improve this in the future

Reply:
Ah yes, this is true. Are some / any of these relevant for mobile?

Reply:
Not that I can see, but we will have to check when we have the bindings working, just something to consider for the future